tjws/src/main/java/Acme/Serve/Serve.java

```diff
@@ -109,6 +109,10 @@
 import java.util.Vector;

```

Comment:
There are a bunch of FINER level logs in the class here; those should be protected with a LOG.isLoggable(Level.FINER) check if any computation is happening to create the arguments for the logs. In particular, it would make sense to have a static DEBUG_ON member, similar to the STREAM_DEBUG being removed, that would take the value of LOG.isLoggable(Level.FINER)